src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunWorkflowTests.kt

```diff
@@ -26,12 +26,4 @@ class RunWorkflowTests : SeleniumTest()
         val url = RequestHelper.webBaseUrl + "/run-workflow/"
         driver.get(url)
     }
-
-    @Test
-    fun `can view run workflow tab`()
-    {
-        val tab = driver.findElement(By.id("run-workflow-tab"))
-        Assertions.assertThat(tab.findElement(By.tagName("h2")).text).isEqualTo("Create workflow")
-        Assertions.assertThat(tab.findElement(By.tagName("p")).text).isEqualTo("Run workflow is coming soon")
-    }
```

Comment:
Not clear why this has been removed

Reply:
This was a test for placeholder. my ticket has overridden its implementation.

Reply:
So this class is now empty? I think there is definitely still value in ensuring that new page/tabs are rendered as part of an in-browser test.